### PR TITLE
ACM-5131 Externalize string for Red Hat OpenShift Virtualization

### PR DESCRIPTION
**Pull request by KevinFCormier**
https://issues.redhat.com/browse/ACM-5131

### CODE REVIEW

File: frontend/public/locales/en/translation.json [hunk 1]
```diff
@@ -332,6 +332,7 @@
...
   "A proxy URL to use for creating HTTP connections outside the cluster. The URL scheme must be HTTP.": "A proxy URL to use for creating HTTP connections outside the cluster. The URL scheme must be HTTP.",
   "A proxy URL to use for creating HTTPS connections outside the cluster. If this is not specified, then httpProxy is used for both HTTP and HTTPS connections.": "A proxy URL to use for creating HTTPS connections outside the cluster. If this is not specified, then httpProxy is used for both HTTP and HTTPS connections.",
+  "A Red Hat OpenShift cluster that is hosted on the Red Hat OpenShift Container Platform in your on-premise datacenter.": "A Red Hat OpenShift cluster that is hosted on the Red Hat OpenShift Container Platform in your on-premise datacenter.",
```
Comment (chenz4027) [inline frontend/public/locales/en/translation.json:335]: Small thing, should be data center instead, from https://issues.redhat.com/browse/ACM-4928

Comment (KevinFCormier) [reply frontend/public/locales/en/translation.json]: Oops - I updated in the code but forgot to update translations. Pushing.

**Review by chenz4027 — APPROVED** [review]
/lgtm

**Review by fxiang1 — APPROVED** [review]
Thanks Kevin for improving this code :)

**Pull request merged**